doc-content/enterprise-only/installation/jws-jdbc-config_proc.adoc

```diff
@@ -0,0 +1,142 @@
+<Context>
+<Resource 
+auth="Container" 
+databaseName="${datasource.dbName}" 
+description="XA Data Source" 
+factory="org.apache.tomcat.jdbc.naming.GenericNamingResourcesFactory" loginTimeout="0" 
+name="xads"
+uniqueName="xads" 
+portNumber="${datasource.port}"
+serverName="${datasource.hostname}" 
+testOnBorrow="false" 
+type="${datasource.class}" 
+url="${datasource.url}" 
+URL="${datasource.url}"
+user="${datasource.username}"
+password="${datasource.password}" 
+driverType="4"
+schema="${datasource.schema}"
+</Context>
+/>
+----
++
+[id='xa_data_source_{context}']
+.XA data source variables
+[cols="40%,60%", options="header"]
+|===
+
+|Variable
+|Description
+
+|`<datasource.dbName>`
+|The name of the database.
+
+|`<datasource.port>`
+|The port number of the database.
+
+|`<datasource.hostname>`
+|The name of the database host.
+
+|`<datasource.class>`
+|`XADataSource` class of JDBC driver.
+
+|`<datasource.url>`
+|The JDBC database connection URL. With some databases, the URL property is `url` and for other databases (for example H2 databases) this property is `URL`.
+
+|`<datasource.username>`
+|User name for the database connection.
+
+|`<datasource.password>`
+|Password for the database connection.
+
+|`<datasource.schema>`
+|The database schema.
+
+|===
+
+.. Configure a pooling data source that relies on the XA data source for creating new connections. In this example, the data source is `poolingXaDs`, `<datasource.username>` is the user name for the database connection, and `<datasource.password>` is the password for the database connection:
++
+[source]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<Context>
+<Resource 
+name="poolingXaDs"
+uniqueName="poolingXaDs"
+auth="Container" 
+description="Pooling XA Data Source" factory="org.jboss.narayana.tomcat.jta.TransactionalDataSourceFactory" testOnBorrow="true" 
+transactionManager="TransactionManager" transactionSynchronizationRegistry="TransactionSynchronizationRegistry" type="javax.sql.XADataSource" 
+username="${datasource.username}" 
+password="${datasource.password}"
+xaDataSource="xads"
+/>
+</Context>
+----
++
+The data source is now available under the `java:comp/env/poolingXaDs` JNDI name and  passes it to the {KIE_SERVER} through the `org.kie.server.persistence.ds` system property as described in the next steps.
++
+[NOTE]
+====
+The pooling data source configuration relies on additional resource that have been previously configured in `context.xml` file in `kie-server` application, specifically `TransactionManager` and `TransactionSynchronizationRegistry`.
+====
+. Configure {KIE_SERVER} to use the data source:
+.. Open one of the following scripts in a text editor:
++
+* For Linux or Unix:
++
+[source]
+----
+JWS_HOME/tomcat/bin/setenv.sh
+----
+* For Windows:
++
+[source]
+----
+JWS_HOME/tomcat/bin/setenv.bat
+----
+
+.. Add the following properties to `CATALINA_OPS` where `<hibernate.dialect>` is the Hibernate dialect for your database:
```

Comment:
Might we want to reference something about what a Hibernate dialect is, or list common ones, or something? I handle this in my document like this: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.1/html/deploying_a_red_hat_process_automation_manager_authoring_environment_on_red_hat_openshift_container_platform/environment-authoring-con#environment-authoring-single-proc search for "Dialect"

Reply:
Done